docs/_templates/relbar.html

```diff
@@ -6,7 +6,7 @@
 <li class="md-tabs__item"><a href="https://gf.com/" class="md-tabs__link"><i class="md-icon">business</i> GlobalFoundries</a></li>
 <li class="md-tabs__item"><a href="https://efabless.com/open_shuttle_program" class="md-tabs__link"><i class="md-icon">precision_manufacturing</i> Shuttle Program</a></li>
 <li class="md-tabs__item"><a href="https://join.skywater.tools/" class="md-tabs__link"><i class="md-icon">chat_bubble</i> Chat</a></li>
```

Comment:
should we also change this to point to the new community?

Reply:
Can you log a new bug and we can fix it?